### ISSUE TITLE
Incorporate work type and subtype updates for some items when doing the migration from hydrus to H2

### ISSUE THREAD

**Issue by amyehodge** [sul-dlss/happy-heron#1389]
Spreadsheet of updates will be linked to from here when it's available. Finishing touches are being put on it now.

**Comment by jcoyne** [comment]
Still waiting on this document to be complete.

**Comment by amyehodge** [comment]
Spreadsheet with new work type/subtype mappings for 1267 items: https://docs.google.com/spreadsheets/d/15yyaBUpY5JUmoCjN2oitVAWF2-uRxe2y1PHMNH_6mSU/edit#gid=1948024480

**Comment by amyehodge** [comment]
@jcoyne There are a couple of reasons.
1. We are not able to directly map some of the existing types to the H2 types, so we would have to assign them to an incorrect type during the migration and then fix them afterwards.
2. We do not want to edit these by hand in the H2 interface after the migration -- and we want to have the changes made before the 1.0 Release. We want the content cleaned up before users get to it again.

If we can do this in a bulk fashion after migrating the content but before the release, i.e. as a separate step, then that would also be satisfactory.

**Comment by amyehodge** [comment]
@jcoyne This hopefully answers your questions.

`"Software/Code" must be "software, multimedia"
"Mixed Materials" must be "mixed material"`

We have used in the spreadsheet the terms that the user chooses from in the UI. If this should be something different for the back end code, then it can be changed. I have done this in the spreadsheet

`the subtype "text" must be "Text"'
I've actually deleted this instance and a few others where "text" (or "Text") was a subtype of "Text", as this is redundant

'"Sound recording" is not a valid subtype of "Music"'
This actually is a valid subtype, but these entries were missing one of the required subtypes. I have corrected that for all of these items by adding a required subtype in the first column, so you shouldn't see this issue again.

Since you didn't mention the other errors called out regarding invalid subtypes, I'll assume you know that all of those actually are valid subtypes and are correcting whatever needs to be corrected to reflect that.
